Sprint-3/2-practice-tdd/count.test.js

```diff
@@ -17,8 +18,16 @@ test("should count multiple occurrences of a character", () => {
   expect(count).toEqual(5);
 });
 
+  
 // Scenario: No Occurrences
 // Given the input string str,
 // And a character char that does not exist within the case-sensitive str,
 // When the function is called with these inputs,
 // Then it should return 0, indicating that no occurrences of the char were found in the case-sensitive str.
+
+test("should return 0 for no occurrences", () => {
```

Comment:
This test is good but it doesn't test different possible function inputs. What if the inputs' types are not strings, the `char `string is longer than the `str` string, or you have an empty string for either of the inputs, etc.? Testing different inputs will show whether your function accounts for different scenarios, which is what makes a function robust.